Missing cog-export.h header in the cogcore_headers

Error: 

```
/work/my-launcher/recipe-sysroot/usr/include/cog/cog-directory-files-handler.h:15:10: fatal error: cog-export.h: No such file or directory
   15 | #include "cog-export.h"
```

--- a/core/meson.build
+++ b/core/meson.build
@@ -23,6 +23,7 @@ cogcore_config_h = configure_file(
 
 cogcore_headers = files(
     'cog.h',
+    'cog-export.h',
     'cog-request-handler.h',
     'cog-directory-files-handler.h',
     'cog-host-routes-handler.h',

